lib/src/migrators/division.dart

```diff
@@ -156,6 +122,44 @@ class _DivisionMigrationVisitor extends MigrationVisitor {
         isDivisionAllowed: true);
   }
 
+  /// Migrates [node] and returns true if it is a new-syntax color function or
+  /// returns false if it is any other function.
+  bool _tryColorFunction(FunctionExpression node) {
+    if (!["rgb", "rgba", "hsl", "hsla"].contains(node.name.asPlain)) {
+      return false;
+    }
```

Comment:
Consider adding empty lines after each `return false` to visually emphasize each distinct check.